### ISSUE TITLE
missing GUID for Balanced

### ISSUE THREAD

**Issue by jredfox** [AaronKelley/PowerMode#2]
the balanced powermode overlay plan has a guid of `00000000-0000-0000-0000-000000000000`

**Comment by bcutter** [comment]
...which is correct this way and the right GUID set by Microsoft. I guess maybe it has changed, anyway the `value` for key `BetterPerformanceGuid` is wrong (`BetterPerformanceGuid` only working as "Balanced" for a system with Modern standby and without classic power plans) as shipped in the current release, you're right.

**Comment by jredfox** [comment]
@bcutter check your pull request there are 4 values now not 3 and they all worked in windows 11 tested two months ago on the latest build

**Comment by bcutter** [comment]
I don't think so. Have a look at https://github.com/AaronKelley/PowerMode/pull/3/files - one line removed, one added. Worked for me. But nevermind, maybe that PR never get's merged anyway.

**Comment by jredfox** [comment]
@bcutter changing the config of the guid just changes the guid to a different one that was valid but doesn't work the same

**Comment by bcutter** [comment]
Do whatever you want :-) I'm gonna leave this thread now.
